fedramp_low_rev_4/sa.sp

```diff
@@ -0,0 +1,22 @@
+benchmark "fedramp_low_rev_4_sa" {
+  title       = benchmark.nist_800_53_rev_4_sa.title
+  description = benchmark.nist_800_53_rev_4_sa.description
```

Comment:
It's great we can re-use descriptions from NIST, were there any sections FedRAMP used that NIST didn't in the Low set?